substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/ImageClassLoader.java

```diff
@@ -69,10 +69,15 @@
 import com.oracle.svm.core.util.InterruptImageBuilding;
 import com.oracle.svm.core.util.VMError;
 import com.oracle.svm.util.ModuleSupport;
-import com.oracle.truffle.api.Truffle;
 
 public final class ImageClassLoader {
 
+    /*
+     * This cannot be a HostedOption because all Subclasses of OptionDescriptors from inside builtin
+     * modules need to be initialized prior to option parsing so that they can be found.
+     */
+    public static final String PROPERTY_IMAGEINCLUDEBUILTINMODULES = "substratevm.ImageIncludeBuiltinModules";
```

Comment:
Should this be named `native-image.ImageIncludeBuiltinModules`? I guess it depends whether this is potentially outward-facing and if we already have properties with either convention.